content/events/ai-workshop.mdx

```diff
@@ -0,0 +1,85 @@
+---
+seo:
+  title: AI Tools 2-day Workshop
+  description: >-
+    2-day AI tools training workshop for developers: Generative AI, prompt
+    engineering, ChatGPT DALLE, GitHub Copilot, and the OpenAI API. Led by
+    experts.
+eventHeader:
+  heroBackground: /images/background/polygonBackground.png
+  imgOverlay: /images/background/SSWDarkMode_Logo.png
```

Comment:
![image](https://github.com/SSWConsulting/SSW.Website/assets/20507092/336066ec-6046-4308-956b-9d3bc15b379b)

Image is very big

Reply:
done - replaced it with a different banner with a smaller logo
Note: image is a placeholder for the real design